src/background.ts

```diff
@@ -94,7 +94,7 @@ async function getTotp(text: string, passphrase: string) {
         if (parameter[0].toLowerCase() === 'secret') {
           secret = parameter[1];
         } else if (parameter[0].toLowerCase() === 'issuer') {
-          issuer = parameter[1];
+          issuer = decodeURIComponent(parameter[1]);
```

Comment:
use try catch here to prevent exception

Reply:
Should I add one here too? Seems kinda pointless otherwise since `label` already contains the same data that I'm performing this on too.

https://github.com/Authenticator-Extension/Authenticator/blob/9e0933d3e116e03c96e648e49b43f7c63c645a4b/src/background.ts#L84

Reply:
Yeah, we should always use try catch to decode URI.

```
try {
  str = decodeURIComponent(str2)
} catch(error) {
  str = str2;
}
```